src/components/Dashboard/CoinInfo.tsx

```diff
@@ -59,10 +60,7 @@ function CoinInfo() {
   const { userXvsBalance } = useMarketsUser();
 
   const handleLink = () => {
-    window.open(
-      `${BASE_BSC_SCAN_URL}/token/${constants.CONTRACT_TOKEN_ADDRESS.xvs.address}?a=${account}`,
-      '_blank',
-    );
+    window.open(`${BASE_BSC_SCAN_URL}/token/${getToken('xvs').address}?a=${account}`, '_blank');
```

Comment:
same construction used second time. what about reusable urlBuilder function for these cases?